.github/workflows/build.yaml

```diff
@@ -338,10 +346,19 @@ jobs:
                           "default") GN_ARGS='target_os="all" is_asan=true enable_host_clang_build=false enable_host_gcc_mbedtls_build=false pw_command_launcher="`pwd`/../scripts/helpers/clang-tidy-launcher.py"';;
```

Comment:
So do we still want the `pw_command_launcher` bit here?

Reply:
No, this was due to merge. Updated, thanks for catching.